app/konnect/api-products/productize-service.md

```diff
@@ -102,19 +102,38 @@ After creating the new version, you will see **Link with a Gateway Service** as
+1. In {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products), select the API product that you created in the previous step.
+1. Click **Add** next to the API product or products you want to publish to the Dev Portal and then click **Finish**. 
+1. Repeat steps 1-3 for any other Dev Portals you want to add the API product to.
+1. In {% konnect_icon api-product %} [**API Products**](https://cloud.konghq.com/api-products), select the API product you added to the Dev Portal. 
+1. Click **Product Versions** in the sidebar.
+1. Click the product version you created previously and in the **Dev Portals** tab, click **Browse Dev Portals** and select the Dev Portals you want to add the product version to. 
+    Product versions are published by default when you select Dev Portals, you can disable publishing while configuring the Dev Portals if you'd like to publish them later.
 
-1. Select the API product that you created in the previous step.
-2. Navigate to the **Actions** menu, then select **publish**. 
-
-Your API product is now consumable by developers from the **Dev Portal**.
+The API product and product versions should now display in the Dev Portals you selected.
 {% endnavtab %}
 {% navtab API %}
-You can publish an API product by issuing a `POST` request to the [`/api-product-versions/`](/konnect/api/api-products/latest/#/API%20Product%20Versions/create-api-product-version) endpoint. Ensure you replace `{ApiProductId}` with the API product ID returned in the previous step.. 
+
+1. Before you publish the API product version, you must first assign the API product to any Dev Portals by issuing a `PATCH` request to the [`/api-products/{id}](/konnect/api/api-products/latest/#/API%20Products/update-api-product) endpoint:
+
+    ```sh
+    curl --request PATCH \
+        --url https://{region}.api.konghq.com/v2/api-products/{ApiProductId} \
+        --header 'Authorization: <personal-access-token>' \
+        --header 'Content-Type: application/json' \
+        --data '{
+        "portal_ids": [
+            "4602b8cc-92fb-4197-b836-3eab20b16147"
+        ]
+    }'
+    ```
+    Be sure to replace `{ApiProductId}`, `name`, and `portal_ids` with your own values.
+
+1. You can publish an API product version by issuing a `POST` request to the [`/api-product-versions/`](/konnect/api/api-products/latest/#/API%20Product%20Versions/create-api-product-version) endpoint. Ensure you replace `{ApiProductId}` with the API product ID returned in the previous step. 
 
     ```sh
     curl -X 'POST' \
-        'https://{region).api.konghq.com/v2/api-products/{ApiProductId}/product-versions' \
+        'https://{region}.api.konghq.com/v2/api-products/{ApiProductId}/product-versions' \
         -H 'accept: application/json' \
         -H 'Authorization: Bearer <personal-access-token>' \
         -H 'Content-Type: application/json' \
```

Comment:
Can we update the summary (lines 148-154) to:

## Summary
In this section, you created an API product and created an API Product version with a linked Gateway Service. You then published both the API Product and API Product version to one or more Dev Portals.

## More information
* [Add API product documentation](/konnect/dev-portal/publish-service/): Learn how to add your API spec and API product documentation to the Dev Portal so your developers can more easily consume your APIs.